drone-localization/core/utils/taskflow.py

```
@@ -0,0 +1,78 @@
+        # postprocess forward
+        if self.upsample_to_original:
+            cls_out,reg_out = self.postprocess(cls_out,reg_out)
+
+        return cls_out, reg_out
+
+    def load_checkpoint(self, checkpoint_path=""):
+        model_keys = set(self.state_dict().keys())
+        ckpt = torch.load(checkpoint_path, map_location="cuda")
+        self.backbone_uav.load_checkpoints(checkpoint_path=checkpoint_path)
+        self.backbone_satellite.load_checkpoints(checkpoint_path=checkpoint_path)
+        missing, unexpected = self.load_state_dict(ckpt, strict=False)
```

Ruff (0.15.12)

[warning] Unpacked variable `missing` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)